app/scripts/components/common/mapbox/layers/raster-timeseries.tsx

```diff
@@ -114,9 +71,7 @@ export function MapLayerRasterTimeseries(props: MapLayerRasterTimeseriesProps) {
   const statuses = useRef<Statuses>({
     global: S_IDLE,
     layer: S_IDLE,
-    // If there's no marker layer (i.e. there's no min zoom extent), set it to
-    // succeeded to be ignored.
-    'zoom-markers': !minZoom ? S_SUCCEEDED : S_IDLE
+    'zoom-markers': S_IDLE
```

Comment:
Given that the markers have nothing to load now, maybe it is a good idea to rename this status? Something like `'bbox-data'`?